docs/getting_started.rst

```diff
@@ -122,7 +155,7 @@ that represents the operator :math:`A = 0.5 * XYZ + 0.5 * YYZ`, we can do:
     A.dim
     A.n_pauli_strings
 
-Just like with ``PauliString`` objects, we can apply ``PauliOp`` objects to a set of quantum states or compute expectation values, as well as arithmetic
+Just like with ``PauliString`` objects, we can apply ``PauliOp`` objects to a set of vectors, or compute expectation values, as well as arithmetic
 operations and dense matrix conversions. Just like with ``PauliString`` objects, we can also convert ``PauliOp`` objects back to dense numpy arrays if we'd like
```

Comment:
`... and dense matrix conversions` - this is discussed twice, in the first and in the second sentences